src/components/Tokens/TokenDetails/TransactionsTable.tsx

```diff
@@ -0,0 +1,118 @@
+import { Table } from 'components/Table'
+import { LinkCell, TextCell } from 'components/Table/Cells'
+import { useActiveLocale } from 'hooks/useActiveLocale'
+import { useMemo } from 'react'
+import { Column } from 'react-table'
+import { useTheme } from 'styled-components'
+import { shortenAddress } from 'utils/addresses'
+import { NumberType, useFormatter } from 'utils/formatNumbers'
+
+import { mockSwapData } from './mockData'
+import { Swap, SwapAction, SwapInOut } from './types'
+import { getLocaleTimeString, getSwapType } from './utils'
+
+enum ColumnHeader {
+  Time = 'Time',
+  Type = 'Type',
+  Amount = 'Amount',
+  For = 'For',
+  USD = 'USD',
+  Maker = 'Maker',
+}
+
+export function TransactionsTable({ referenceTokenAddress }: { referenceTokenAddress: string }) {
+  const locale = useActiveLocale()
+  const { formatNumber } = useFormatter()
+  const theme = useTheme()
+
+  const columns: Column<Swap>[] = useMemo(() => {
+    const getColor = (action: SwapAction) => (action === SwapAction.Buy ? theme.success : theme.critical)
+    return [
+      {
+        Header: ColumnHeader.Time,
+        accessor: (swap) => swap,
+        Cell: ({ value }: { value: { timestamp: number; transactionHash: string } }) => (
+          <LinkCell url={`https://etherscan.io/tx/${value.transactionHash}`} justifyContent="left">
```

Comment:
use the helper function [getExplorerLink](https://github.com/Uniswap/interface/blob/main/src/utils/getExplorerLink.ts)

Reply:
Nice, using this now